runway/module/terraform.py

```diff
@@ -414,7 +415,14 @@ def cleanup_dot_terraform(self) -> None:
                 self.logger.debug("directory retained: %s", child)
                 continue
             self.logger.debug("removing: %s", child)
-            send2trash(str(child))  # does not support Path objects
+            if Path.exists(child):
+                try:
+                    Path.unlink(child)
+                except IsADirectoryError:
+                    shutil.rmtree(child)
+                except OSError:
+                    self.logger.warning("unable to remove %s, sending to .Trash", child)
+                    send2trash(child)  # does not support Path objects
```

Comment:
```suggestion
            if child.is_dir():
                shutil.rmtree(child, ignore_errors=True)
            else:
                child.unlink(missing_ok=True)
```

- `child` is already a `Path` object so no need to pass it into `Path.exists` or `Path.unlink`
- check if its a directory first rather than catching an error raised if it is a directory
- the arguments passed in the suggestion ignore _most_ errors that would be raised except if its a file. in that case it only accounts for the file already being deleted
- removes the need for `send2trash` as a dependency (it can be removed with `poetry remove send2trash`)

Reply:
sounds good, thanks!